sdk/identity/identity/src/msal/nodeFlows/nodeCommon.ts

```diff
@@ -20,15 +20,35 @@ import {
   msalToPublic,
   publicToMsal
 } from "../utils";
+import { TokenCachePersistenceOptions } from "./tokenCachePersistenceOptions";
 
 /**
  * Union of the constructor parameters that all MSAL flow types for Node.
  * @internal
  */
 export interface MsalNodeOptions extends MsalFlowOptions {
+  tokenCachePersistenceOptions?: TokenCachePersistenceOptions;
   tokenCredentialOptions: TokenCredentialOptions;
 }
 
+/**
+ * The current persistence provider, undefined by default.
+ * @internal
+ */
+let persistenceProvider:
+  | ((options?: TokenCachePersistenceOptions) => Promise<msalCommon.ICachePlugin>)
+  | undefined = undefined;
+
+/**
+ * An object that allows setting the persistence provider.
+ * @internal
+ */
+export const msalNodeFlowCacheControl = {
+  set persistence(pluginProvider: Exclude<typeof persistenceProvider, undefined>) {
```

Comment:
same feedback, setter props feel strange when the property could just be a function that returns void